test/cppsuite/test_harness/workload_tracking.h

```diff
@@ -0,0 +1,82 @@
+#ifndef WORKLOAD_TRACKING_H
```

Comment:
Why we need this?

Reply:
#ifndef and #define are known as header guards. Their primary purpose is to prevent C++ header files from being included multiple times.